### ISSUE TITLE
Plantillas disponibles en el sistema multiurl

### ISSUE THREAD

**Issue by contidos** [chamilo/chamilo-lms#4654]
Actualmente en Chamilo 1.11.x se puede crear una plantilla de documento personalizada pero, en caso de estar usando un sistema multiurl, cualquier plantilla que se cree en un portal hijo, aparecerá también en el portal madre y en el resto de hijos. Esto no resulta práctico porque si el portal A tiene un modelo de plantillas que no quiere que el portal B vea, ambos compartirán todo el repositorio de plantillas. 

![screenshot-www apacampus com-2023 03 22-10_12_50](https://user-images.githubusercontent.com/30044847/226855619-1f41a825-f8ae-49f4-bc8c-0dc4a950bf1e.png)

Esto es debido a que no existe un campo "url" en la tabla system_template o una tabla de relación entre la tabla de system_template y access_url

Una solución ideal sería que, estando activo el modo multiurl, la creación de plantillas fuese un sistema propio de cada portal.

**Comment by ywarnier** [comment]
Sí, efectivamente, no existe esta funcionalidad. Si no es urgente, preferiría ponerla en Chamilo 2.0 porque el sistema de plantillas en sí es relativamente mal estructurado en Chamilo 1.

**Comment by contidos** [comment]
Sí, no es urgente para nada.
